docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -0,0 +1,71 @@
+```markdown
+
+```
+
+# Step 6 - Enabling account registration
+
+We are trying to let our friends write their bike trips directly into our website. We would like to let them create Rides. As an Administrator of the website you could create the Users yourself but In order to let them create their own content, first, you want to let them create their own accounts.
+
+## Add the registration possible
+
+From the Admin Panel, go to the _Roles_ management screen, click on _Anonymous_ Role name link to edit it.
+
+![step_6_role_mgmt_screen](/home/shaim/www/developer-documentation/docs/tutorials/platform_beginner/img/step_6_role_mgmt_screen.png)
+
+For the Anonymous User, you will add the _User/Register_ policy. This will enable the access to the Register form for everyone on your website.
+![step6_admin_anonymous_policies](/home/shaim/www/developer-documentation/docs/tutorials/platform_beginner/img/step6_admin_anonymous_policies.png)
+
+
+
+!!! note
+
+​    See the documentation about [Registering new users to learn how to assign a group](https://doc.ezplatform.com/en/1.7/guide/user_generated_content/#registering-new-users) to the newly registered Users.
+
+
+
+Then go to the URL: http://127.0.0.1/register and create a new user.
+
+The Users are created in the _Guest accounts_ group, and the User you have just created will have the Roles used by the _Guest accounts_ User group.
+Now, we have 2 options:
+
+* **Add the right to create Rides to _Guest accounts_.** This will enable the Ride creation for whoever having registered on Go Bike website. In this case, you may not want to give the Publish Content permission.
+* **Create a group dedicated to our Go Bike Members** and give to this group the right to create Rides. Then you will have to add the Users to the Go Bike Members group so they can benefit from the Roles of the group. In this case, once the User is "_validated_" as trusted, they will be able to create and publish their own content.
+
+In this Tutorial, you will use the second option.
+
+## Create a User Group for the Go Bike Members
+
+From the Admin Panel in the _Users_ screen, click on the *Create* button and _filter by Group_ by checking "_Users_" to see only the users-related Content Types. Create a User Group named `Go Bike Members`.
+
+## Create a Folder for the Rides from the Go Bike Members
+
+In the Content creation screen, select _Folder_ and create a new Folder named `members-rides ` into the Folder `all-rides` already existing. The Go Bike Members will have the access rights in creation mode only in this Folder.
```

Comment:
In this case I'm going to insist on my earlier comment:

"Member Rides" and "All Rides" to be consistent with the earlier steps: create a Folder named "Member Rides" in the existing "All Rides" Folder.

"access rights in creation mode only" is unclear to me.